howso/engine/trainee.py

```diff
@@ -22,6 +22,12 @@
 from howso.client.protocols import ProjectClient
 from howso.direct import HowsoDirectClient
 from howso.engine.client import get_client
+from howso.openapi.models import (
+    Metrics,
```

Comment:
If we are moving these out from under TYPE_CHECKING, they should be added to the existing `howso.openapi.models` import.

Reply:
Good eye. Fixed.